internal/cmd/stack/environment.go

```diff
@@ -73,6 +119,89 @@ func setVar(cliCtx *cli.Context) error {
 	return nil
 }
 
+func (e *listEnvCommand) listEnv(cliCtx *cli.Context) error {
+	outputFormat, err := cmd.GetOutputFormat(cliCtx)
+	if err != nil {
+		return err
+	}
+
+	stackID := cliCtx.String(flagStackID.Name)
+
+	var query listEnvQuery
+	variables := map[string]interface{}{
+		"stack": graphql.ID(stackID),
+	}
+
+	if err := authenticated.Client.Query(cliCtx.Context, &query, variables); err != nil {
+		return err
+	}
+
+	var outputs []listEnvElementOutput
+	for _, attachedContext := range query.Stack.AttachedContexts {
+		for _, config := range attachedContext.Config {
+			outputs = append(outputs, config.toConfigElementOutput(attachedContext.ContextName))
+		}
+	}
+	for _, config := range query.Stack.RuntimeConfig {
+		var contextName *string
+		if config.Context != nil {
+			contextName = &config.Context.ContextName
+		}
+		outputs = append(outputs, config.Element.toConfigElementOutput(contextName))
+	}
+
+	switch outputFormat {
+	case cmd.OutputFormatTable:
+		return e.showOutputsTable(outputs)
+	case cmd.OutputFormatJSON:
+		return e.showOutputsJSON(outputs)
+	}
+
+	return fmt.Errorf("unknown output format: %v", outputFormat)
+}
+
+func (e *listEnvCommand) showOutputsTable(outputs []listEnvElementOutput) error {
+	tableData := [][]string{{"Name", "Type", "Value", "File Mode"}}
```

Comment:
Can you add a column header for the "Context" column please?

Reply:
Thanks, that's now fixed.